### PR TITLE
[distribution_list]

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
[event_mass_mailing]
[mass_mailing_distribution_list]
[CHG][ADD]
Translations

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
:+1:

File: mass_mailing_distribution_list/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#. module: mass_mailing_distribution_list
+msgid "Outgoing Mails"
+msgstr "Courriels sortant"
+
+#: field:distribution.list,partner_path:0
+#. module: mass_mailing_distribution_list
+msgid "Partner Path"
+msgstr "Chemin vers contact"
+
+#: view:mail.mass_mailing:mass_mailing_distribution_list.view_mail_mass_mailing_form
```
Comment (anthony-muschang) [inline mass_mailing_distribution_list/i18n/fr.po]: Translation on a domain here. Should be remove.

File: mass_mailing_distribution_list/i18n/mass_mailing_distribution_list.pot [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#. module: mass_mailing_distribution_list
+#: model:ir.model,name:mass_mailing_distribution_list.model_mail_mail
+msgid "Outgoing Mails"
+msgstr ""
+
+#. module: mass_mailing_distribution_list
+#: field:distribution.list,partner_path:0
+msgid "Partner Path"
+msgstr ""
+
```
Comment (anthony-muschang) [inline mass_mailing_distribution_list/i18n/mass_mailing_distribution_list.pot:140]: Translation on a domain here. Should be remove.

File: event_mass_mailing/i18n/fr.po [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+#. module: event_mass_mailing
+msgid "Send Invitation"
+msgstr "Envoi d'invitation"
+
+#: view:event.event:event_mass_mailing.view_event_form
+#. module: event_mass_mailing
+msgid "Sent Date"
+msgstr "Date d'envoi"
+
+#: view:event.event:event_mass_mailing.view_event_form
```
Comment (anthony-muschang) [inline event_mass_mailing/i18n/fr.po]: Translation on a domain here. Should be remove.

File: event_mass_mailing/i18n/event_multiple_registration.pot [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+#. module: event_mass_mailing
+#: view:event.event:event_mass_mailing.view_event_form
+msgid "Send Invitation"
+msgstr ""
+
+#. module: event_mass_mailing
+#: view:event.event:event_mass_mailing.view_event_form
+msgid "Sent Date"
+msgstr ""
+
```
Comment (anthony-muschang) [inline event_mass_mailing/i18n/event_multiple_registration.pot:124]: Translation on a domain here. Should be remove.

**Comment by JonathanNEMRY** [comment]
:+1:

**Comment by anthony-muschang** [comment]
:+1:

**Pull request merged**